kubernetes/check-pods.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash -e
+
+pod_label=$1
+statefulset=$2
```

Comment:
Instead of requiring this parameter maybe it would be better to check if there's either a pod or stateful set with that name?

If you want to keep the parameter please use a value that's more self-explanatory. When you call the script as `./check-pods.sh verifier true` it's hard to tell what this `true` means. Maybe add an option like `--stateful-set`? Or `--type=statefulset|pod`?

Reply:
Done.